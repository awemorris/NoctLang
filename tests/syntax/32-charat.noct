func main() {
    var a = "ABCあいう";
    for (i in 0..7) {
        print(a->charAt(i));
    }
}
