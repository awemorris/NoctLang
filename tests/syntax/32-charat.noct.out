A
B
C
あ
い
う

